nservicebus/aws/local-development.md

```diff
@@ -0,0 +1,40 @@
+---
+title: AWS local development using LocalStack
+summary: How to use LocalStack for your development needs when using Amazon SQS Transport and DynamoDB Persistence
+reviewed: 2024-10-29
+---
+
+[LocalStack](https://www.localstack.cloud/) is a tool to develop and test your AWS applications locally reducing development time and increasing product velocity.
+
+> [!NOTE]
+> Refer to the [official LocalStack documentation](https://docs.localstack.cloud/getting-started/installation/) to learn how to install and run it locally.
+
+To configure an NServiceBus endpoint to connect to LocalStack instead of connecting to AWS, the AWS endpoint URL must to be set to the address of the LocalStack instance.
+
+There are several ways you can achieve that.
```

Comment:
```suggestion
To configure an NServiceBus endpoint to connect to LocalStack instead of connecting to AWS, the AWS endpoint URL must to be set to the address of the LocalStack instance. There are several ways you can achieve that.
```